topic-controller/resources/openshift-template.yaml

```diff
@@ -24,6 +24,13 @@ rules:
   - create
   - patch
   - update
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - get
```

Comment:
If you are creating events, perhaps it should contain also `create`? Or only `create`?

Reply:
This is what I pointed out in the previous review, maybe it's just a typo ?